inc/saiacl.h

```diff
@@ -998,10 +998,19 @@ typedef enum _sai_acl_table_attr_t
      */
     SAI_ACL_TABLE_ATTR_FIELD_IPV6_NEXT_HEADER,
 
+    /**
+     * @brief Bridge Type
+     *
+     * @type bool
+     * @flags CREATE_ONLY
+     * @default false
+     */
+    SAI_ACL_TABLE_ATTR_FIELD_BRIDGE_TYPE,
```

Comment:
Please highlight the use case. Is the idea to match all the packets that are classified to either 1Q Bridge or 1D bridge? Or we want to match bridge OID. Also we might want to add qualifiers like bridge OID.

Reply:
@atitjain The requirement for this pull request is not to match on the bridge OID, but just to match packets classified on a .1Q bridge or on any .1D bridge. We want to classify packets at the switch bind point level, hence this match field is needed.

Yes it would be useful to add bridge OID and also a new bindpoint for .1D bridge similar to vlan. But the requirement is to also differentiate at the switch bind point level.

Reply:
Yes that's the idea.

Reply:
@lguohan @atitjain Can this pull request be merged ? We had discussed this OCP call and i don't see any additional comments from the OCP members.